ci-scripts/linux/tahoma-build.sh

```diff
@@ -26,4 +21,4 @@ cmake ../sources  $CANON_FLAG \
     -DWITH_GPHOTO2:BOOL=ON \
     -DWITH_SYSTEM_SUPERLU=ON
 
-make -j "$parallel"
+make -j$(($(nproc) < 2 ? 1 : $(nproc) - 1))
```

Comment:
This is wrong. We dont want to build libTIFF in this script anymore, since you are trying to build it as part of CMake. The change I proposed should be in the FindTIFF.cmake file.